src/main/java/core/basesyntax/service/impl/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import java.io.File;
+
+public class Main {
+    public static void main(String[] args) {
+        ReportService reportService = new ReportServiceImpl();
+        File dataFile = new File("input.csv");
+        reportService.sendReport(dataFile);
```

Comment:
in the main method, you should create every single service separately and call their method sequentially. For example:

```
ReaderService readerService = new ReaderServiceImpl();
List<String> rows = readerService.readFromFile(INPUT_FILE);
....
String report = reportService.createReport();
writerService.writeToFile(report, OUTPUT_FILE);
```